.env.example

```diff
@@ -21,3 +21,6 @@ DB_USER=
 DB_PASSWORD=
 DB_HOST=
 DB_PORT=
+
+# API Secret Token
+API_Secret_Token=
```

Comment:
Better keep environment variables in all CAPS format.

Reply:
done